### ISSUE TITLE
Cannot create hit (error : missing ad_url config var)

### ISSUE THREAD

**Issue by fspinar** [NYUCCL/psiTurk#523]
Hi all,

I am trying to use create hit command but got the following error : 

Response(Operation: create_hit | Status: error | Exception: PsiturkException: missing ad_url config var `ad_url_domain`)

I am editing config.txt file and assigning my heroku app's url to ad_url_domain, but I still get the same error.

Any suggestions ? 

Thanks.

**Comment by lchen64** [comment]
I have the same problem, and noticed the get_ad_url function in psiturk_config.py is not used apparently. Can this part be explained? Thanks

**Comment by deargle** [comment]
Not enough information!

**Comment by lchen64** [comment]
Yeah I realized that too late, sorry.

So the problem I am having is that my heroku ad url is set, and read
correctly by psiTurk, but the app page just says "There is nothing here yet"

Earlier I didn't have this problem, but when MTurkers tried to do my task,
they finished it but were not able to hit Submit afterwards. Could this be
an issue with them needing to reload the page first? I saw a forum post
about the submit part being finicky since its a JavaScript Ajax request
embedded in HTML. Thanks!
